devtools-ci-index.yaml

```diff
@@ -655,6 +663,63 @@
             $ssh_cmd -t "cd payload-tests/integration-tests && /bin/bash cico_run_tests.sh"
     <<: *job_template_defaults
 
+- job-template:
+    name: '{ci_project}-devtools-test-end-to-end-prod-fabric8-analytics'
```

Comment:
{ci_project}-devtools-test-end-to-end-prod-fabric8-analytics
would expand into:
devtools-devtools-test-end-to-end-prod-fabric8-analytics

You want this to be:
{ci_project}-test-end-to-end-prod-fabric8-analytics